runner/README.md

```diff
@@ -38,7 +41,8 @@ This provides a general enough, isolated and ephemeral execution environment. Ea
 
 ### Using Docker to easily boot up a local cluster
 - Install Docker 18+ and start it up
-- Run `docker run -it --name bob-queue -p 5672:5672 -p 15672:15672 rabbitmq:3-management-alpine` to run the latest management enabled RabbitMQ instance on port `5672` and the admin control on port `15672`. The default credentials are `guest:guest`.
+- Run `docker run -it --name bob-queue -p 5672:5672 -p 15672:15672 -p 5552:5552 rabbitmq:3-management-alpine` to run the latest management enabled RabbitMQ instance on port `5672`, the streams interface on port `5552` and the admin control on port `15672`. The default credentials are `guest:guest`.
```

Comment:
lets use the command like this:
```sh
docker run -it --name bob-queue -p 5552:5552 -p 5672:5672 -p 15672:15672 -e RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS='-rabbitmq_stream advertised_host localhost' --entrypoint sh rabbitmq:management-alpine -c 'rabbitmq-plugins enable --offline rabbitmq_stream && rabbitmq-server'
```
instead of the two step process